client/packages/cli/README.md

```diff
@@ -109,15 +109,77 @@ Options:
 - **-t, --timeout \<timeout\>**: Timeout in seconds for waiting for events from the chain. Default timeout is 30 seconds
 - **-x, --export**: Export extrinsic data to a file
 
+### estimate-gas-fee
 
-### estimate
+The `estimate-gas-fee` command is used to estimate the gas fee required for an execution
 
-The `estimate` command is used to estimate the max reward with a given target amount.
+Usage:
+
+```bash
+yarn cli estimate-gas-fee [options]
```

Comment:
We use pnpm instead of yarn in cli

Reply:
Fixed; see commit [801c3a8](https://github.com/t3rn/t3rn/pull/1242/commits/801c3a8be8d93b0759177f8d6f8efc28a99a1829).